### PR TITLE
[16.0][ADD] product_supplierinfo_import_by_barcode: New module

### PR DESCRIPTION
**Pull request by chienandalu**
cc @Tecnativa TT49604

please review @pedrobaeza @carlosdauden

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: product_supplierinfo_import_by_barcode/models/product_supplierinfo_import_template.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Tecnativa - David Vidal
```
Comment (carlosdauden) [inline product_supplierinfo_import_by_barcode/models/product_supplierinfo_import_template.py]: would be 2024 or it is old code?
```suggestion
# Copyright 2024 Tecnativa - David Vidal
```

File: product_supplierinfo_import_by_barcode/models/product_template.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Copyright 2021 Tecnativa - David Vidal
```
Comment (carlosdauden) [inline product_supplierinfo_import_by_barcode/models/product_template.py]:
```suggestion
# Copyright 2024 Tecnativa - David Vidal
```

File: product_supplierinfo_import_by_barcode/wizards/product_supplierinfo_import.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
+# Copyright 2021 Tecnativa - David Vidal
```
Comment (carlosdauden) [inline product_supplierinfo_import_by_barcode/wizards/product_supplierinfo_import.py]:
```suggestion
# Copyright 2024 Tecnativa - David Vidal
```

**Review by carlosdauden — APPROVED** [review]
Minor comments

File: product_supplierinfo_import_by_barcode/models/product_supplierinfo_import_template.py (hunk 1 above)
Comment (chienandalu) [reply product_supplierinfo_import_by_barcode/models/product_supplierinfo_import_template.py]: Nah, just copy-pasted... :sweat_smile: fixed

**Pull request merged**